go.mod

```diff
@@ -642,7 +642,7 @@ require (
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
-	github.com/DataDog/agent-payload/v5 v5.0.137
+	github.com/DataDog/agent-payload/v5 v5.0.138-0.20241128131042-e8f5da68c088
```

Comment:
will lock version when https://github.com/DataDog/agent-payload/pull/355 is merged